style.md

```diff
@@ -2593,6 +2593,7 @@ type Client struct {
 Embedding should provide tangible benefit, like adding or augmenting
 functionality in a semantically-appropriate way. It should do this with zero
 adverse user-facing effects (see also: [Avoid Embedding Types in Public Structs]).
+Generally, mutexes should not be embedded.
```

Comment:
I'm also okay with removing this line - reading the description provides the same information.

However, given the prevalence of embedded mutexes, I thought it might be worth calling it out explicitly.

Reply:
We previously called out mutexes above, so moving it here instead is fine - we can always remove it later if we decide it's not adding any value.